<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchPinList</class>
 <widget class="QWidget" name="BatchPinList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelBasePath">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayoutCtrl">
       <item>
        <widget class="QPushButton" name="proceedButton">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Pin selection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
